src/utils/proto.ts

```diff
@@ -66,7 +65,8 @@ export const queryCall = async (
     arg,
   });
 
-  if (queryResponse.status == QueryResponseStatus.Rejected) {
+  // TODO: Cannot access ambient const enums when the '--isolatedModules' flag is provided. - we cannot use QueryResponseStatus.Rejected
```

Comment:
I'd delete this TODO altogether. The proto will be deprecated at some point anyway, so likely this TODO will never be addressed.

Reply:
Cool!